src/css/style.css

```diff
@@ -0,0 +1,835 @@
+/*
+=========
+  RESET
+=========
+*/
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+
+/*
+===============
+
+COLOR VARIABLES
+
+===============
+*/
+
+:root {
+  --l-gray: gainsboro;
+  --gray: #808080;
+  --red-orange: #f42;
+  --nav-color: #aaa;
+  --l-orange: #fff4f2;
+  --pink: #ffebe7;
+  --l-pink: #fffaf8;
+  --pink-gradient:
+    linear-gradient(
+      145deg,
+      rgba(236, 106, 108, 0.906582701439951) 37%
+    );
+  --black-gradient:
+    linear-gradient(
+      0deg,
+      rgba(0, 0, 0, 0.83375356978729) 33%,
+      rgba(0, 0, 0, 0.45840342973126746) 47%,
+      rgba(0, 0, 0, 0) 64%
+    );
+}
+
+#wrapper {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  max-width: 100vw;
+  margin: 0 auto;
+  grid-auto-rows: minmax(50px, auto);
+  grid-template-areas:
+    "nav nav"
+    "header header"
+    "main main"
+    "footer footer";
+}
+
+header {
+  grid-area: header;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+main {
+  grid-area: main;
+}
+
+nav {
+  grid-area: nav;
+  font-size: 12px;
+  z-index: 1;
+  position: relative;
+  max-width: 1440px;
+}
+
+.head-logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background:
+    linear-gradient(
+      118deg,
+      rgba(211, 9, 137, 1) 39%,
+      rgba(231, 36, 90, 1) 62%,
+      rgba(255, 68, 34, 1) 90%
+    );
+  height: 47px;
+  width: 100px;
+  z-index: 2;
+  position: absolute;
+  position: fixed;
+}
+
+.head-logo img {
+  object-fit: contain;
+  z-index: 2;
+  width: 70px;
+  height: 70px;
+}
+
+.nav-bars {
+  top: 0;
+  position: fixed;
+  min-width: 100%;
+  max-width: 1440px;
+  background-color: black;
+}
+
+.flex {
+  display: flex;
+}
+
+nav li {
+  margin: 0 10px;
+  display: none;
+}
+
+.right-nav {
+  justify-content: flex-end;
+  float: right;
+}
+
+.left-nav {
+  float: left;
+}
+
+.top-articles a {
+  color: black;
+}
+
+.contact a {
+  font-size: 8px;
+  font-weight: 600;
+  margin: 0 5px;
+  color: var(--gray);
+}
+
+.bottom-nav a {
+  color: var(--nav-color);
+}
+
+.flex-box a {
+  font-weight: 700;
+}
+
+.funding a {
+  color: black;
+}
+
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your ` color: black;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.color-black{
   color: black;
}
```
**Html file**

```
<div class="color-black">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_